.changeset/breezy-flowers-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-app-api': patch
+---
+
+Deprecate the `featureDiscoveryServiceFactory` in favor of using `discoveryFeatureLoader` instead.
```

Comment:
Best point to the location of the `discoveryFeatureLoader`, same in the `plugin-api` changeset.

Reply:
Done